packages/storybook/package.json

```diff
@@ -12,14 +12,14 @@
     "build-storybook-local": "rm -rf storybook-static && yarn build-storybook && cd storybook-static && npx http-server"
   },
   "dependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
     "@mui/icons-material": "^5.15.19",
     "@mui/material": "^5.15.19",
-    "esbuild": "0.21.5",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-markdown": "^9.0.1"
+    "react-markdown": "^9.0.1",
+    "vite-plugin-svgr": "^4.2.0"
```

Comment:
moving this from devDeps to deps was an intentional change?

Reply:
Yes, was not required in devdeps and it is in deps